mlir/lib/Gradient/IR/GradientOps.cpp

```diff
@@ -22,12 +22,12 @@
 #include "Gradient/Utils/CompDiffArgIndices.h"
 #include "Gradient/Utils/GradientShape.h"
 
-#define GET_OP_CLASSES
-#include "Gradient/IR/GradientOps.cpp.inc"
-
 using namespace mlir;
 using namespace catalyst::gradient;
 
+#define GET_OP_CLASSES
+#include "Gradient/IR/GradientOps.cpp.inc"
+
 // Gradient input checker
```

Comment:
Just curious, was this deliberate (i.e. do the generated op classes depend on the imported namespaces)?

Reply:
Yup, the generated classes contain references to `MemRefType` and `ShapedType` that are unknown without the `using namespace mlir;`.